ui.apps/src/main/content/jcr_root/apps/core/cif/components/renderconditions/pagetemplate/pagetemplate.jsp

```diff
@@ -0,0 +1,48 @@
+<%--
```

Comment:
Is there a reason why this is a JSP and not a servlet? We already have the bulk of the business logic in the `TemplateRenderCondition` class, we might as well make it a servlet.

Reply:
All the documentation and existing components/examples for `granite:rendercondition` were done like that, so I just followed the "guidelines". A JSP is actually a servlet ;-)

Reply:
>A JSP is actually a servlet ;-)

You should talk to some people about this, they'll have a different opinion 😆 

Moving the logic into a bundle protects us from backwards compatibility issues. A JSP is considered to be public API (starting with 6.4 IIRC) so once we release it it's basically locked. I know we are not part of quickstart, but we still have to maintain BC.